examples/matching-engine/tests/transaction.rs

```diff
@@ -180,11 +180,7 @@ async fn single_transaction() {
             amount
         );
     }
-    for (owner, amount) in [
-        (admin_account, None),
-        (owner_a, Some(Amount::ZERO)),
-        (owner_b, None),
-    ] {
+    for (owner, amount) in [(admin_account, None), (owner_a, None), (owner_b, None)] {
```

Comment:
Minor: we could skip the `amount` binding here, because it's always `None` :man_shrugging:

Reply:
Right, and merge the two loops. :+1: